src/main/java/com/google/cloud/tools/appengine/configuration/AppEngineWebXmlProjectStageConfiguration.java

```diff
@@ -127,8 +127,20 @@ public String getRuntime() {
     return runtime;
   }
 
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  /**
+   * @deprecated Use
+   *     newBuilder().sourceDirectory(sourceDirectory).stagingDirectory(stagingDirectory) instead.
+   */
+  @Deprecated
   public static Builder builder(Path sourceDirectory, Path stagingDirectory) {
-    return new Builder(sourceDirectory, stagingDirectory);
+    Builder builder = new Builder();
+    builder.sourceDirectory(sourceDirectory);
+    builder.stagingDirectory(stagingDirectory);
+    return builder;
```

Comment:
Chain them like `return new Builder().sourceDirectory(...).stagingDirectory(..)`?

Reply:
Agreed.  Should be consistent (see `AppYamlProjectStageConfiguration`)

Reply:
done, although this style is my last favorite part of the builder pattern as "fluent" interfaces prioritize writeability over readability and debuggability.